### ISSUE TITLE
Add live website URL to repo description and README.

### ISSUE THREAD

**Issue by CharlesStover** [north-seattle/website#19]
As a web developer but non-Python user, I find it difficult to contribute to this project, because it appears to be entirely based on a Python package. I would like to see an "end result" of the build process for the existing repo as a way to tie what I am reading in the code to what is being published live. Otherwise, my intuition is to scrap Python as a whole as a web development foundation -- I am unable to see what is being _done_ currently.

**Comment by cwinters8** [comment]
Hey @CharlesStover, there isn't a live site yet for the project. We made some progress on it during the hackathon tonight, so hopefully something will go live soon. Stay tuned!

**Comment by timothycrosley** [comment]
Hi @CharlesStover and @cwinters8, 

there is now a live website here: https://northseattle.dev/, and the system has been updated to continuously deploy changes to the website destination as soon as they are merged into the master branch. 

The project uses Pelican (a Python powered framework) for turning standard Markdown files into an HTML website. However, It shouldn't require any Python knowledge for usage, as the content is written in Markdown, and the themes etc are written in HTML + JavaScript + CSS. I've built off @cwinters8 work to explain this process, and how to make changes and see them locally (without Python knowledge, but with Python installed) if desired: https://github.com/north-seattle/website#northseattledev.

Thanks!

~Timothy
